NOM_visualizations/python/nom_data.ipynb

```diff
@@ -5,12 +5,13 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    import nmdc_api_utilities
</pre><p>Just FYI: This may not be sufficient in Google Colab, and you may have to add a cell to install this module first (at least that's what we had to do in <a href="https://github.com/microbiomedata/nmdc_notebooks/blob/main/NEON_soil_metadata/python/neon_soil_metadata_visual_exploration.ipynb" target="_blank">https://github.com/microbiomedata/nmdc_notebooks/blob/main/NEON_soil_metadata/python/neon_soil_metadata_visual_exploration.ipynb</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/149/discussion/#comment-2004491784'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>